R/jags_model.R

```diff
@@ -0,0 +1,39 @@
+# nolint start
+
+# LogisticNormal ----
+
+jags_model_data_ln <- function() {
```

Comment:
in terms of naming, we can here consider a new prefix `jm_`  and then spend more characters on the rest.
e.g. `jm_lik_logistic` (likelihood of logistic model)
note that in the likelihood here it is independent of normal or log-normal prior.

Reply:
I used `data` in the name to somehow indicated this is data model and also to match `datamodel` slot in model related class. I see other logistic models have different formula, so probably would be worth to keep `jm_lik_logistic_normal`, and of course the same will be used to log-normal.